charts/self-hosted/values.yaml

```diff
@@ -13,11 +13,12 @@ global:
     rootKey: 997ec52a856f0911442d1fccc308a408
 
   directory:
-    version: 0.21.13
+    version: 0.21.15
     rootKey: c96fe48da0735cf77fdba50134f52c43
 
   postgresql:
     localServerEnabled: true
+    existingSecret: "{{ .Release.Name }}-postgresql"
     auth:
       postgresHost: "{{ .Release.Name }}-postgresql"
       postgresUser: postgres
```

Comment:
Confirming that, in the default case, the charts aserto-directory & bitnami will look for the key `postgres-password` in the secret `{{ .Release.Name }}-postgresql` and fail to find it if it doesn't exist? Essentially, `existingSecret` is not optional? Doesn't that make the `postgresPassword` key below this line dead code?

Reply:
I believe `postgresPassword` will still get used by the Bitnami chart to set the initial db password and create the credential secret.  I'll update the docs to clarify this.

Reply:
But you're right it's no longer used after initial db creation.